app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,28 @@
-<h2>Log in</h2>
+<div class="logIn-body">
+  <h2>Log in</h2>
 
-<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
-
-  <div class="field">
-    <%= f.label :password %><br />
-    <%= f.password_field :password, autocomplete: "current-password" %>
-  </div>
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name), html: { class: "login-form" }) do |f| %>
+    <div class="field">
+      <%= f.label :email, class: "placeholder-label" %>
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "input" %>
+    </div>
 
-  <% if devise_mapping.rememberable? %>
     <div class="field">
-      <%= f.check_box :remember_me %>
-      <%= f.label :remember_me %>
+      <%= f.label :password, class: "placeholder-label" %>
+      <%= f.password_field :password, autocomplete: "current-password", class: "input" %>
     </div>
-  <% end %>
 
-  <div class="actions">
-    <%= f.submit "Log in" %>
-  </div>
-<% end %>
+    <% if devise_mapping.rememberable? %>
+      <div class="field">
+        <%= f.check_box :remember_me %>
```

Comment:
- Good job done so far but as the previous reviewer said, please ensure that the registration page also matches the given design wireframe. 👍 👍 
<img width="944" alt="wrawrawraw" src="https://github.com/evansnyamekye/Blog-application/assets/73607512/ce1785eb-ab98-4838-80a4-55092f465fc0">